brax/training/agents/ppo/networks.py

```diff
@@ -68,7 +68,9 @@ def make_ppo_networks(
     .identity_observation_preprocessor,
     policy_hidden_layer_sizes: Sequence[int] = (32,) * 4,
     value_hidden_layer_sizes: Sequence[int] = (256,) * 5,
-    activation: networks.ActivationFn = linen.swish) -> PPONetworks:
+    activation: networks.ActivationFn = linen.swish,
+    policy_state_key: str = 'state',
+    value_state_key: str = 'state') -> PPONetworks:
```

Comment:
What would you pass in as `value_state_key` if you wanted `state` and `state_privileged` in the value network?

Reply:
In my current implementation, the env defines privileged state as the union of state + additional privileged information.